required/latex-lab/latex-lab-math.dtx

```diff
@@ -310,11 +310,25 @@
       {
         \@@_process:nn { math } {#1} % $
 % fairly simple this one
-        \tagmcend %end P-chunk, in code: \tag_mc_end_push:
-        \@kernel@math@begin
-        #1 $
-        \@kernel@math@end
-        \tagmcbegin{}  % restart P-chunk (whatsits in pdftex)
+%    \end{macrocode}
+% We do not want math tagging in fakemath or when measuring, 
+% so we imitate the test inside
+% \cs{@@_process:nn} for now, see https://github.com/latex3/tagging-project/issues/5
```

Comment:
As it says, OK for now!  

Further work is needed here, particularly for some edge cases, but, for now:  Looks good to me.